From b79f7b3660251350ad6483b131fb2e36abd0ff68 Mon Sep 17 00:00:00 2001
From: "sukil.hong" <sukil.hong@lge.com>
Date: Mon, 17 Oct 2022 14:24:43 +0900
Subject: [PATCH] Fix compilation with lttng-ust >= 2.13

:Release Notes:
Fix compilation with lttng-ust >= 2.13

:Detailed Notes:
|  error: #include expects "FILENAME" or <FILENAME>
|  1140 | #include LTTNG_UST_TRACEPOINT_INCLUDE
|       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~

long deprecated TRACEPOINT_INCLUDE_FILE macro was
finally removed in 2.13.5, see:
https://github.com/lttng/lttng-ust/commit/41858e2b6e8
https://github.com/lttng/lttng-ust/commit/d2966b4b0b2

:Testing Performed:
Build Test

:QA Notes:
N/A

:Issues Addressed:
[WRO-12611] LTTng tracing is not working

Change-Id: Iaa2e7bcf068ac584302c8562202959a0bf8b9878
Upstream-Status: Pending
---
 performancelog/plugin/pmtrace_performancelog_provider.h | 6 +++---
 pmtrace/plugin/pmtrace_qmlplugin_provider.h             | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/performancelog/plugin/pmtrace_performancelog_provider.h b/performancelog/plugin/pmtrace_performancelog_provider.h
index cac89d4..1ab811c 100644
--- a/performancelog/plugin/pmtrace_performancelog_provider.h
+++ b/performancelog/plugin/pmtrace_performancelog_provider.h
@@ -1,4 +1,4 @@
-// Copyright (c) 2013-2018 LG Electronics, Inc.
+// Copyright (c) 2013-2022 LG Electronics, Inc.
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
@@ -17,8 +17,8 @@
 #undef TRACEPOINT_PROVIDER
 #define TRACEPOINT_PROVIDER pmtrace_performancelog
 
-#undef TRACEPOINT_INCLUDE_FILE
-#define TRACEPOINT_INCLUDE_FILE ./pmtrace_performancelog_provider.h
+#undef TRACEPOINT_INCLUDE
+#define TRACEPOINT_INCLUDE "./pmtrace_performancelog_provider.h"
 
 #ifdef __cplusplus
 extern "C"{
diff --git a/pmtrace/plugin/pmtrace_qmlplugin_provider.h b/pmtrace/plugin/pmtrace_qmlplugin_provider.h
index b7dd291..19ca59a 100644
--- a/pmtrace/plugin/pmtrace_qmlplugin_provider.h
+++ b/pmtrace/plugin/pmtrace_qmlplugin_provider.h
@@ -1,4 +1,4 @@
-// Copyright (c) 2013-2018 LG Electronics, Inc.
+// Copyright (c) 2013-2022 LG Electronics, Inc.
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
@@ -17,8 +17,8 @@
 #undef TRACEPOINT_PROVIDER
 #define TRACEPOINT_PROVIDER pmtrace_qmlplugin
 
-#undef TRACEPOINT_INCLUDE_FILE
-#define TRACEPOINT_INCLUDE_FILE ./pmtrace_qmlplugin_provider.h
+#undef TRACEPOINT_INCLUDE
+#define TRACEPOINT_INCLUDE "./pmtrace_qmlplugin_provider.h"
 
 #ifdef __cplusplus
 extern "C"{
-- 
2.37.1

